src/components/reactTasks/displayComments.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import _ from 'lodash'
+
+class DisplayComments extends React.Component {
+
+  // Render a list of the amount of comments passed in commmentsMax prop
+  render() {
+    return (
+      <ul>
+        {_.map(this.props.comments, (comment, index) => {
```

Comment:
probs better to do it in two steps. Get `this.props.commentMax` comments. Then map them to `<li>` and stuff.

`_.take` or something?